src/main/java/net/snowflake/client/core/FileCacheManager.java

```diff
@@ -317,4 +317,9 @@ private static long fileCreationTime(File targetFile) {
     }
     return -1;
   }
+
+  @SnowflakeJdbcInternalApi
```

Comment:
it's package scope method in package scope class - it does not need the annotation, it's already internal

Reply:
Right, removed it.